bcachefs/bcachefs.py

```diff
@@ -263,3 +278,11 @@ def __init__(self, fs: _Bcachefs):
 
     def __next__(self):
         return DirEnt(*super(BcachefsIterDirEnt, self).__next__())
+
+
+class BcachefsIterInode(BcachefsIter):
```

Comment:
Could probably go before `BcachefsIterDirEnt()` since `INODE_TYPE == 1` and `DIRENT_TYPE == 2`

Reply:
done